src/test/java/teammates/test/cases/action/StudentProfilePictureActionTest.java

```diff
@@ -335,9 +335,14 @@ protected StudentProfilePictureAction getAction(String... params) {
         return (StudentProfilePictureAction) gaeSimulation.getActionObject(getActionUri(), params);
     }
 
+    @Test
     @Override
     protected void testAccessControl() throws Exception {
-        //TODO: implement this
+        String[] submissionParams = new String[] {
+                Const.ParamsNames.USER_ID, account.googleId,
+                Const.ParamsNames.BLOB_KEY, account.studentProfile.pictureKey
+        };
+        verifyAnyRegisteredUserCanAccess(submissionParams);
```

Comment:
Access control for this action seems to be well tested by existing methods. Move relevant methods under `testExecuteAndPostProcess` to `testAccessControl`.